server/createExpressHandler.ts

```diff
@@ -0,0 +1,26 @@
+import 'dotenv/config';
+import { Request, Response } from 'express';
+import { ServerlessContext, ServerlessFunction } from './types';
+import Twilio from 'twilio';
+
+const { ACCOUNT_SID, TWILIO_API_KEY_SID, TWILIO_API_KEY_SECRET, VIDEO_IDENTITY } = process.env;
+
+const twilioClient = Twilio(TWILIO_API_KEY_SID, TWILIO_API_KEY_SECRET, {
+  accountSid: ACCOUNT_SID,
+});
+
```

Comment:
I think we need a way to let users know if they are using bad credentials. Right now, if we pass a bad api key in the .env file, then the app seems to work, and there's just a signaling error in the UI. It looks like a problem with the end user's connection, when it's not. 

One thing we could do, is just prevent this dev server from running if the creds are bad. We could something like this here here (beware of the pseudocode):
```
// Make request just to make sure that the api key that was supplied is valid
twilioClient.tokens.create().catch((err) => {
  if (/* is 401 error */) {
    console.error('Unable to authenticate...')
  } else {
    console.error(err.usefulErrorMessage)
  }

  process.exit(1);
});
```

With this change, we'll have to pass the `-k` flag to `concurrently` in package.json, so that the React dev server is also stopped if the credentials are bad. This seems like a decent approach since nothing with run without good credentials:
```
"start": "concurrently -k npm:server npm:dev",
```

What do you think about this approach?

Reply:
I think that sounds great! I would rather have a user know upfront that the the application is not set up correctly. This way they will not go through the entire flow of the application only to find that there is some vague error somewhere.